dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/common/hibernate/HibernateIdentifiableObjectStore.java

```diff
@@ -1039,7 +1039,7 @@ public Date getLastUpdated()
     public List<Function<Root<T>, Predicate>> getSharingPredicates( CriteriaBuilder builder, UserInfo userInfo,
         CurrentUserGroupInfo groupInfo, String access )
     {
-        if ( !sharingEnabled( userInfo ) || userInfo == null || groupInfo == null )
+        if ( !sharingEnabled( userInfo ) || userInfo == null || groupInfo == null || userInfo.isSuper() )
```

Comment:
Could we centralize this check a bit more? Four components in a conditional statement is bit too many I think.